src/DDoSX/Client.php

```diff
@@ -71,6 +111,24 @@ public function ssls()
         return (new SslClient($this->httpClient))->auth($this->token);
     }
 
+    /**
+     * Return the AclGeoIpClient instance
+     * @return AclGeoIpClient
+     */
+    public function aclGeoIps()
+    {
+        return (new AclGeoIpClient($this->httpClient))->auth($this->token);
+    }
+
+    /**
+     * Return the HstsClient instance
+     * @return HstsClient
+     */
+    public function hsts()
```

Comment:
we're adding separate clients for cnd/waf rules here, but haven't split up hsts rules - I'm wondering if that was deliberate